src/app/pages/datasets/components/dataset-node/dataset-encryption-cell/dataset-encryption-cell.component.html

```diff
@@ -1,15 +1,26 @@
 <div class="encryption-description" *ngIf="!dataset.encrypted; else encryptedCell">{{ 'Unencrypted' | translate }}</div>
 <ng-template #encryptedCell>
+
   <ng-container *ngIf="dataset.locked; else unlockedCell">
-    <mat-icon>{{ isEncryptionRoot ? 'lock' : 'lock_outline' }}</mat-icon>
+    
```

Comment:
What was wrong with old code here?

Reply:
The lock icons we need are not in the standard mat-icon set. I had to get them from the mdi icon set.